easybuild/tools/package/utilities.py

```diff
@@ -173,20 +193,20 @@ def __init__(self):
         if sel_pns in avail_pns:
             self.pns = avail_pns[sel_pns]()
         else:
-            raise EasyBuildError("Selected package naming scheme %s could not be found in %s",
-                                 sel_pns, avail_pns.keys())
+            raise EasyBuildError("Selected package naming scheme %s could not "
+                                 "be found in %s", sel_pns, avail_pns.keys())
```

Comment:
It was a line length issue, both pep8 and pylint complain about it. 

If we want to make a standard longer line I'm fine with that.